projects/plugins/jetpack/extensions/blocks/ai-assistant/lib/prompt/index.ts

```diff
@@ -145,9 +156,7 @@ function getCorrectSpellingPrompt( {
 	return [
 		{
 			role,
-			content: `Repeat the text delimited with ${ delimiter }, without the delimiter, correcting any spelling and grammar mistakes directly in the text without providing feedback about the corrections, keeping the language of the text: ${ getDelimitedContent(
-				content
-			) }`,
+			content: `Repeat the following text, but correcting any spelling and grammar mistakes directly in the text without providing feedback about the corrections, keeping the language of the text:\n\n${ content }`,
```

Comment:
Why remove the delimiter?

Reply:
I tried to make it simpler, at least for the POC. Adding the delimiter requires extra processing that I wanted to skip when testing the approach. Also, I slightly remember that the place where the app detectes the delimiter is not optimal.

Being said that the AI responds very well afaics with the line breaks (I'm aware of the tokens)